source/blog/2018-07-27-the-ember-times-issue-57.md

```diff
@@ -12,7 +12,12 @@ Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/27/the-embe
 
 ---
 
-## [SECTION]()
+## [Ember Inspector - Help shape the future!](https://github.com/emberjs/ember-inspector)
+We're pushing hard to make Ember Inspector the world class dev tools you deserve, and would love to get your input and help!
```

Comment:
should this read "dev tool" (no 's')

Reply:
Looks like its one word now, devtools.
- https://developers.google.com/web/tools/chrome-devtools
- https://developer.mozilla.org/son/docs/Tools